Docs for close-connection TP action (#1520)
ngrok/ngrok-docs: traffic-policy/actions/close-connection/behavior.mdx

## Behavior

> If this action is executed, connection to the endpoint will be closed immediately. As an example, This will be useful in dealing with DDOS attacks.

Reviewer: ```suggestion
When this action is executed, connection to the endpoint is closed immediately.
This can be useful for dealing with DDoS attacks.
```
